src/client-services/github.client-service.test.tsx

```diff
@@ -0,0 +1,69 @@
+import axios, { AxiosInstance, AxiosResponse } from 'axios';
+import {
+  axiosConfig,
+  getReposList,
+  getRepoInfo,
+} from '../client-services/github.client-service';
+
+jest.mock('axios');
+
+describe('Github client service test', () => {
+  const accessToken = 'testToken';
+  const mockResponse: AxiosResponse = {
+    data: [{ name: 'openbc-web' }],
+    headers: {
+      Authorization: `token ${accessToken}`,
+      'Content-Type': 'application/json',
+    },
+    config: {},
+    headers: {},
+    request: {},
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    process.env.GITHUB_API_KEY = 'testToken';
+  });
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+    delete process.env.GITHUB_API_KEY;
+  });
+
+  describe('axiosConfig', () => {
+    it('should return an Axios instance', async () => {
+      axios.create.mockResolvedValueOnce(mockResponse);
+      const axiosInstance: AxiosInstance = await axiosConfig();
+
+      expect(axiosInstance).toEqual(mockResponse);
+    });
+  });
+
+  describe('getReposList', () => {
+    it('should get the list of repositories', async () => {
+      axios.create.mockReturnValueOnce({
```

Comment:
Change to the following

`
      axios.create = jest.fn().mockReturnValueOnce({
        get: jest.fn().mockResolvedValueOnce(mockResponse),
      });
`

Reply:
Will change